modules/apps/layout/layout-type-controller/layout-type-controller-panel/src/main/resources/META-INF/resources/layout/view/panel.jsp

```diff
@@ -18,7 +18,10 @@
 
 <c:choose>
 	<c:when test="<%= !themeDisplay.isStatePopUp() %>">
-		<aui:container class="lfr-panel-page" id="main-content">
+		<clay:container
```

Comment:
Similarly here

Have we checked this change is correct?

```html
<div class="container-fluid lfr-panel-page">
```

```html
<div class="container-fluid container-fluid-max-xl lfr-panel-page />
```

Being this a panel... I'm not so sure about this one

Reply:
As above https://github.com/eduardoallegrini/liferay-portal/pull/64#discussion_r417185555